### ISSUE TITLE
"Someone is using an unknown session" warning is confusing and lacks clear call-to-action

### ISSUE THREAD

**Issue by vertigo220** [vector-im/element-meta#950]
"Someone is using an unknown session" with red shield in DMs

#### What would you like to do?
Have this be more clear

#### Why would you like to do it?
So it's easier for people to figure out what they're doing and be secure

#### How would you like to achieve it?
Instead of just saying "Someone is using an unknown session" have it be specific about who the issue is with, what the issue is, and how to resolve it. My friend and I had gone through and set up all of the security and privacy settings (we verified all our sessions and set up secure backup, cross-signing, and cryptography), we'd verified each other with the emojis, and we both showed to ourselves as being fully verified and secure. We both looked for a few minutes and couldn't find the problem and then just gave up. Only after I did more digging later and we checked again another day did we find the somewhat buried emoji verification which we had to do again. I'm guessing it's because when we first started using it we accidentally created a room instead of a DM and verified each other there, or maybe because we did it in Nheko and needed to do it separately in Element, but it was confusing. As Matrix is attempting to get more user-friendly, I see this as an area in need of improvement. I'd suggest having the message both be more clear and also take people to where they need to go to fix the problem.

**Comment by daniellekirkwood** [comment]
@pmaier1 I believe this is already on your backlog/ in your view to work on. An update from you would remove the needs-product label and this issue could be linked to whichever epic you're working from :)

**Comment by ajkessel** [comment]
Until the UI/UX is improved here, is there a specific set of steps we can point people to who experience this warning?

**Comment by richvdh** [comment]
> Until the UI/UX is improved here, is there a specific set of steps we can point people to who experience this warning?

People can't fix the problem on their own: the problem is that another user needs to fix it. Find the person with the red ~shield~ star in the user list and tell them to go and verify their devices.

@mxandreas what are our plans for this shield and text? Do you think we can get rid of it altogether once we enable "exclude insecure devices"?

**Comment by richvdh** [comment]
Related: https://github.com/element-hq/element-web/issues/30644

**Comment by mxandreas** [comment]
> Do you think we can get rid of it altogether once we enable "exclude insecure devices"?

Yes, once insecure devices get excluded, they are no longer a problem and thus the shield becomes also unnecessary.

**Comment by richvdh** [comment]
> Yes, once insecure devices get excluded, they are no longer a problem and thus the shield becomes also unnecessary.

This change is now tracked at https://github.com/element-hq/element-web/issues/30661.

It's also worth noting that, since this issue was opened, Element-Web's behaviour has changed in that the shield is only shown if a *verified* user has unverified devices, which means it is relatively unlikely you will see the shield.

Given those factors, I don't think there's any point keeping this issue open.

**Comment by IzzySoft** [comment]
Having a clear call-to-action on this, would have saved us ~5h of try-and-error yesterday…
